src/components/building/buildingnav.js

```diff
@@ -0,0 +1,131 @@
+import styled from 'styled-components';
+
+import {bindActionCreators} from 'redux';
+import {useDispatch, useSelector} from 'react-redux';
+import {actionCreators} from '../../state';
+
+import {checkRescource} from '../../util/checkResources';
+
+import {gameBuildingsTypes} from '../../util/gamedatas/gameBuildingsTypes';
+import {timeBuilder} from '../../util/BuildingTime';
+
+export const Buildingnav = ({currentUserBuildings, selectedBuilding}) => {
+  const {setSelectedBuilding} = bindActionCreators(actionCreators, useDispatch());
+  const currentUserResources = useSelector(state => state.currentUserResources);
+  const currentUserBuildingInProgress = useSelector(state => state.currentUserBuildingInProgress);
+
+  return (
+    <StyledSection>
+      {gameBuildingsTypes.map(gameBuildingType => {
+        const currentUserBuilding = currentUserBuildings.find(
+          currentUserBuildings => currentUserBuildings.buildingId === gameBuildingType.id
+        );
+
+        const selectedBuildID = selectedBuilding.buildingId;
+
+        let borderColor = 'black';
+
+        if (gameBuildingType.id === selectedBuildID) {
+          borderColor = 'green';
+        }
```

Comment:
You could use a ternary here:
```suggestion
        const borderColor = gameBuildingType.id === selectedBuildID ? 'green' : 'black';
```

Reply:
done